starters/svelte-kit-scss/src/routes/(examples)/api-example/+page.server.ts

```diff
@@ -1,13 +1,11 @@
+import { getGreeting } from '$lib/server';
 import type { PageServerLoad } from './$types';
 
-export const load: PageServerLoad = async ({ fetch }) => {
-  const endpoint = 'https://api.starter.dev/.netlify/functions/server/hello?greeting=This Dot Labs!';
-
-  const response = await fetch(endpoint);
-
-  const message: string = await response.text();
+export const load: PageServerLoad = async ({ url }) => {
+  const greeting = url.searchParams.get('greeting');
+  const message = await getGreeting(greeting || 'This Dot Labs!');
```

Comment:
```suggestion
  const message = await getGreeting(greeting ?? 'This Dot Labs!');
```

If we use the nullish coalescing operator (??) here, the user can send an empty string and 0, if they wanted 🤷‍♂️ 

Probably not a big deal, but it would give them a way to see the default `Hello, there` message from the API